### DISCUSSION TITLE
Difference in number of IDs identified

### DISCUSSION THREAD

**Discussion by ABRai11** [vdemichev/DiaNN#849] [Q&A] [answered]
Hi,

I had a question about the DIA data analysis.
I searched a DIA data of 3 samples together which identified less IDs compared to searching the same data individually. Can anyone help me to find out the reason?

Thank you in advance.

**Comment by vdemichev** [comment, marked as answer]
This is likely the result of global precursor FDR filtering and is perfectly fine. You can use the main report of DIA-NN to get full control of how the data is being filtered.

**Comment by ABRai11** [comment]
Ok, Thank you.
